scripts/document.sh

```diff
@@ -40,6 +40,7 @@ mkdir -p /tmp/mbnavigation/
 README=/tmp/mbnavigation/README.md
 cp docs/cover.md "${README}"
 perl -pi -e "s/\\$\\{MINOR_VERSION\\}/${MINOR_VERSION}/" "${README}"
+perl -pi -e "s/\\$\\{SHORT_VERSION\\}/${SHORT_VERSION}/" "${README}"
```

Comment:
I wonder if jazzy supports this, looks like an obvious use case.

Reply:
It seems that `{{module_version}}` in the templates should work as we provide `${SHORT_VERSION}` to `--module-version`. But not sure if it applies to the README file.
The custom variables are not supported for themes: _(check issue 566 in Jazzy)_